4-storage/redshift-storage/sql/atomic-def.sql

```diff
@@ -9,10 +9,10 @@
 -- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
 --
--- Version:     0.10.0
+-- Version:     0.11.0
```

Comment:
should this be 0.10.1, to mirror postgres?

Reply:
I'm not too sure on the semantics of this. For Redshift we need to create an entirely new table whereas for Postgres we can just run an `ALTER COLUMN`.

Reply:
I think Redshift never had anything in PATCH except 0, even when only ALTER TABLE was enough. I still think that these versions are preferable. At least if we're going to change something - let's change Postgres.

Reply:
@chuwy so should we change postgres?

Reply:
I'm in doubt as I also don't know what are semantics of these versions (it isn't SchemaVer for sure). But I think yes - let's change postgres.